### PR TITLE
[IMP] marketing: sms essentials automated action v16

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=2057358&cids=3&menu_id=4722&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@ksc-odoo 

This is a change to lines 179-184 for version-specific changes to the SMS automated action PR. 

Can you review it when you have a moment?

Thanks :+1: 

Tim

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 1]
```diff
@@ -176,6 +175,41 @@ is visualized as different graphs and charts, which can be sorted and grouped in
    :align: center
    :alt: Reporting page in SMS Marketing.
 
+.. tip::
+   SMS messages can be sent using automated actions in Odoo. Odoo *Studio* is required to use
+   automated actions. To install it, go to :menuselection:`Apps --> Search...` bar and search for
+   `studio`. If it is not already installed, click :guilabel:`Install`. Adding the *Studio*
+   application upgrades the subscription status to *Custom*, which increases the cost. Consult
+   `support <https://www.odoo.com/contactus>`_ or reach out to the database's customers success
+   manager with any questions on upgrading.
```
Comment (ksc-odoo) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:184]:
```suggestion
   SMS messages can be sent using automated actions in Odoo. Odoo *Studio* is required to use
   automated actions.
   
   To install Odoo *Studio*, go to :menuselection:`Apps application`. Then, using the
   :guilabel:`Search...` bar, and search for `studio`.
   
   If it is not already installed, click :guilabel:`Install`.
   
   Adding the *Studio* application upgrades the subscription status to *Custom*, which increases the
   cost. Consult `support <https://www.odoo.com/contactus>`_, or reach out to the database's
   customer success manager, with any questions on upgrading.
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 2]
```diff
@@ -176,6 +175,41 @@ is visualized as different graphs and charts, which can be sorted and grouped in
...
+
+   Under :guilabel:`Action To Do` drop-down field, select :guilabel:`Send SMS Text Message`. Next,
+   set the :guilabel:`SMS Template`, and choose whether the SMS message should be logged as a note,
+   by ticking the box next to :guilabel:`Log as Note`.
+
+   .. image:: sms_essentials/automated-action-sms.png
+      :align: center
+      :alt: Automated action template with action to do, SMS template and log as note highlighted.
+
+   Ensure the :guilabel:`Active` toggle is set to *on* for the automated action to run.
```
Comment (ksc-odoo) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:209]: might be nice to place this between the 'Model' and 'Trigger' instructions just so the reader can follow along with how items/fields appear on the form, sequentially

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 3]
```diff
@@ -176,6 +175,41 @@ is visualized as different graphs and charts, which can be sorted and grouped in
...
+   `studio`. If it is not already installed, click :guilabel:`Install`. Adding the *Studio*
+   application upgrades the subscription status to *Custom*, which increases the cost. Consult
+   `support <https://www.odoo.com/contactus>`_ or reach out to the database's customers success
+   manager with any questions on upgrading.
+
+   To use automated actions, navigate in :ref:`developer mode <developer-mode>`, to
+   :menuselection:`Settings app --> Technical menu --> Automation section --> Automated Actions`.
+   Then, click :guilabel:`New` to create a new action.
+
+   Enter an :guilabel:`Action Name` and select a :guilabel:`Model` to implement this action on. Set
```
Comment (ksc-odoo) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:190]:
```suggestion
   Enter an :guilabel:`Action Name`, and select a :guilabel:`Model` to implement this action on. Set
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 4]
```diff
@@ -176,6 +175,41 @@ is visualized as different graphs and charts, which can be sorted and grouped in
...
+   the :guilabel:`Trigger` to either :guilabel:`On Creation`, :guilabel:`On Update`, :guilabel:`On
+   Creation & Update`, :guilabel:`On Deletion`, :guilabel:`Based on Form Modification`, or
+   :guilabel:`Based on Timed Condition`.
+
+   Based on the selection for the :guilabel:`Trigger`, additional fields may populate.
+
+   Under the :guilabel:`Apply on` field a record filter using a domain can be created. Ensure a
```
Comment (ksc-odoo) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:197]:
```suggestion
   Under the :guilabel:`Apply on` field, a record filter using a domain can be created. Ensure a
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst [hunk 5]
```diff
@@ -176,6 +175,41 @@ is visualized as different graphs and charts, which can be sorted and grouped in
...
+   model has been selected before setting any domains on the :guilabel:`Apply on` field. Click
+   :guilabel:`Edit Domain` to set record parameters.
+
+   Under :guilabel:`Action To Do` drop-down field, select :guilabel:`Send SMS Text Message`. Next,
+   set the :guilabel:`SMS Template`, and choose whether the SMS message should be logged as a note,
+   by ticking the box next to :guilabel:`Log as Note`.
```
Comment (ksc-odoo) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:203]:
```suggestion
   by ticking the checkbox next to :guilabel:`Log as Note`.
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst (hunk 2 above)
Comment (ksc-odoo) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:209]:
```suggestion
   Ensure the :guilabel:`Active` toggle is set to *on*, represented by a full-green switch, in order for the automated action to run.
```

**Review by ksc-odoo — APPROVED** [review]
@tiku-odoo -- just finished my Final Review of this ... nicely done. approving now. only a small amount of minor adjustments that require your attention. Once those are implemented, you can feel free to tag this for Tech Review. 👍

**Comment by tiku-odoo** [comment]
@samueljlieber 

This PR is ready for your review when you have a moment. Thanks!

Lines 178-219

This is for all 16 versions.

Tim

**Review by samueljlieber — APPROVED** [review]
Looks good to me @tiku-odoo, nice job! Approving and FWP to only 16 versions.
..
@robodoo r+
@fw-bot up to saas-16.4

**Pull request closed without merging**